ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -334,12 +337,15 @@ def _get_disk_by_id(worker):
 
 
 @retry(AssertionError, 120, 10, 1)
-def verify_pvs_created(expected_pvs, storageclass):
+def verify_pvs_created(expected_pvs, storageclass, exact_count_pvs=True):
```

Comment:
@vavuthu in which situations would the PVs count be higher than requested?

Reply:
@ebenahar If BM server has 15 disks, then 15 PV's created which leads verify_pvs_created failed